hoot-core/src/main/cpp/hoot/core/schema/OverwriteTagMerger.h

```diff
@@ -54,9 +56,16 @@ class OverwriteTagMerger : public TagMerger
   // leave empty to avoid duplicate tag mergers displayed by the info command
   virtual QString getDescription() const { return ""; }
 
+  virtual void setConfiguration(const Settings& conf);
+
+  void setOverwriteExcludeTagKeys(const QStringList& overwriteExcludeTagKeys)
+  { _overwriteExcludeTagKeys = overwriteExcludeTagKeys; }
+
 protected:
 
   bool _swap;
+  // keys of tags not to be overwritten
+  QStringList _overwriteExcludeTagKeys;
```

Comment:
Only implemented this for the overwrite logic path and not tag generalize or averaging, since only overwrite was required (does anyone ever use generalize or average??).